public/app/features/dashboard/components/TransformationsEditor/TransformationEditor.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useContext } from 'react';
+import { css } from 'emotion';
+import { CustomScrollbar, Icon, JSONFormatter, ThemeContext } from '@grafana/ui';
+import { GrafanaTheme, DataFrame } from '@grafana/data';
+
+interface TransformationEditorProps {
+  name: string;
+  description: string;
+  editor?: JSX.Element;
+  input: DataFrame[];
+  output?: DataFrame[];
+  debugMode?: boolean;
+}
+
+export const TransformationEditor = ({ editor, input, output, debugMode }: TransformationEditorProps) => {
+  const theme = useContext(ThemeContext);
+  const styles = getStyles(theme);
+
+  return (
+    <div>
+      <div className={styles.editor}>
+        {editor}
+        {debugMode && (
+          <div className={styles.debugWrapper}>
+            <div className={styles.debug}>
+              <div className={styles.debugTitle}>Input</div>
+              <div className={styles.debugJson}>
+                <CustomScrollbar
+                  className={css`
+                    height: 100%;
+                  `}
+                >
+                  <JSONFormatter json={input} />
+                </CustomScrollbar>
```

Comment:
not sure about these internal scrollbars, they feel unnecessary when the outer container (tab) is already scrollable